plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -255,6 +258,11 @@ public IcebergTableHandle getTableHandle(ConnectorSession session, SchemaTableNa
                 name.getTableName(),
                 name.getTableType(),
                 snapshotId,
+                getFileFormat(table),
+                getDataPath(table.location()),
+                serializeToBytes(table.schema()),
+                serializeToBytes(table.spec()),
```

Comment:
I think there already was an idea to add schema to IcebergTableHandle and it was rejected (?) for some reason.

@phd3 do you remember?